### PR TITLE
fix(cli): changed validation for the signup data and voice credit data

### PR DESCRIPTION
**Pull request by crisgarner**
# Description

changed validation for the signup data and voice credit data

## Confirmation

- [x] I have read and understand MACI's [contributor guidelines](https://maci.pse.dev/docs/contributing) and [code of conduct](https://maci.pse.dev/docs/contributing/code-of-conduct).
- [x] I have read and understand MACI's [GitHub processes](https://github.com/privacy-scaling-explorations/maci/discussions/847).
- [x] I have read and understand MACI's [testing guide](https://maci.pse.dev/docs/testing).

### CODE REVIEW

File: cli/ts/commands/signup.ts [hunk 1]
```diff
@@ -1,8 +1,8 @@
+import { ContractTransactionReceipt, isBytesLike } from "ethers";
```
Comment (0xmad) [inline cli/ts/commands/signup.ts:1]:
```suggestion
import { type ContractTransactionReceipt, isBytesLike } from "ethers";
```

**Review by 0xmad — COMMENTED** [review]
@crisgarner thanks, just one comment

**Comment by crisgarner** [comment]
> @crisgarner thanks, just one comment

Thanks! added it

**Review by 0xmad — APPROVED** [review]
@crisgarner thanks!

**Pull request merged**